app/assets/stylesheets/home.css

```diff
@@ -1,3 +1,90 @@
 .right {
     float: right;
 }
+
+div.centerHead{
+  float:center;
+
+  background-color: green;
+  width: 30%;
+  /*margin:auto;*/
+  margin:  10px auto;
+  border: 5px solid #ccc;
+}
+
+/* Style the tab */
+div.tab {
+  overflow: hidden;
+  border: 5px solid #ccc;
+  background-color: green;
+  /*background-color: #f1f1f1;*/
+}
+
+/* Style the buttons inside the tab */
+div.tab button {
+  background-color: inherit;
+  float: left;
+  border: none;
+  outline: none;
+  cursor: pointer;
+  padding: 14px 16px;
+  transition: 0.3s;
+  color:#ffffff;
+}
+
+/* Change background color of buttons on hover */
+div.tab button:hover {
+  background-color: #ddd;
+}
+
+/* Create an active/current tablink class */
+div.tab button.active {
+  background-color: #ccc;
+}
+
+/* Style the tab content */
+.tabcontent {
+  display: none;
+  padding: 6px 12px;
+  border: 1px solid #ccc;
+  border-top: none;
+}
+
+div.about{
+  border: 5px solid #ccc;
+  background-color: green;
+  width: 50%;
+  padding: 0px 10px;
+  align:center;
+  float:left;
+  margin:10px 0px;
+}
+
+h1{
```

Comment:
This might be one situation where I actually think type selectors are good, you're not adding unnecessary complexity and it fits, the only thing is it'll impact literally every h1 on the site. So instead put this inside the new homepage_header above